atlp-rwanda/Dev-Pulse-FN
# chore(node version): updating node version

#### What does this PR do?
> Update node version
#### Description of Task to be completed?
- update to use node version 12.x
- uninstalling unused packaes
#### How should this be manually tested?
1.  clone this repo
1. Make sure you are using the latest version on node and
1.  create a `.env` file and copy all variables from `.env.example`
1. run `yarn install`
1.  go to `http://localhost:5000/` 
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
#### Questions:
N/A
